checkstyle/suppressions.xml

```diff
@@ -208,8 +208,8 @@
     <suppress checks="MethodLength"
               files="RocksDBWindowStoreTest.java"/>
 
-    <suppress checks="MemberName"
-              files="StreamsPartitionAssignorTest.java"/>
+    <suppress checks="MemberName|ConstantName"
```

Comment:
I assume this is because of `private static final TaskId task0_0`, why not just call it `TASK_0_0`, etc?

Reply:
Hm, I just removed the `ConstantName` suppression and checkstyle passed. Maybe I was doing something weird that got cleaned up.
I'm guessing that wasn't really your question, though. I can't speak to why the author of `StreamsPartitionAssignor` decided to violate/suppress checkstyle back in the day, but I personally find the tests easier to read with the `task0_0` naming convention. I'm happy to negotiate though

Reply:
It's a bit annoying because different tests in the assignment package all need the same set of variables, but have different naming conventions. Some just chose to make them non-static rather than suppress the checkstyle MemberName warning.
Seeing as every test file starts with a long number of declarations of the same thing, I'd prefer to just move them all (and any shared util methods) to a separate `AssignmentTestUtils` file. I stopped myself from doing that in this PR, but if/when we do consolidate them we can discuss which of the several naming conventions to go with

Reply:
As I understand it, the checkstyle rule is enforcing the official code style for the AK project, so we shouldn't just play jazz. But I can get behind your desire to standardize the test style, at least within the module. If you prefer not to uppercase the field names, you can just drop `static` but keep `final`, right? In the grand scheme, making it static is just saving a miniscule amount of RAM during tests.

Reply:
Unfortunately I also have gripes about the non-static naming rules as well, as `task0_0` is still not allowed (underscore not allowed). Maybe I'm just being unreasonable here...

Reply:
(Not trying to continue the argument, I conformed all names to the oppressive checkstyle rules in the end 🙂)